scst-store/use-external-database.hbs.md

```diff
@@ -1,43 +1,56 @@
 # Use external PostgreSQL database for Supply Chain Security Tools - Store
 
-This topic describes how you can configure and use your external PostgreSQL database for Supply Chain Security Tools (SCST) - Store.
+For production deployments, it is recommended to use an external Postgres database rather the one packaged with the Tanzu Application Platform.  This will allow you to manage it via best practices and process that your organization have established.  This topic describes how you can configure your TAP deployment to use an external database.
 
 ## <a id='prereqExtrenalDB'></a>Prerequisites
 
-- Set up your external PostgreSQL database. After the database instance starts, retrieve the following information:
+- Gather the connection information for the external PostgreSQL database: 
 
-   1. Database Instance Endpoint
-   2. Main User name
-   3. Main Password
-   4. Database Name
+   1. Database Instance Connect Endpoint
+   2. (Optional if TLS is enabled and certificate is self signed) CA Certificate that signed the certificate
+   3. Database Credentials
+   4. Name of the datbase to use for AMR/MDS
```

Comment:
Acronyms should be spelled out on the first use.

Reply:
Based on pervious call out of AMR not being in 1.5 and 1.6, just remove AMR/MDS entirely.